content/docs/1.6.0/advanced-resources/os-distro-specific/talos-linux-support.md

```diff
@@ -0,0 +1,55 @@
+---
+title:  Talos Linux Support
+weight: 5
+---
+
+Before Longhorn installation on a Talos Linux cluster, specific configurations are required.
+
+- [Installation Of System Extensions](#installation-of-system-extensions)
+- [Data Path Mounts](#data-path-mounts)
+- [References](#references)
+
+## Installation Of System Extensions
+
+Longhorn depends on `siderolabs/iscsi-tools` and `siderolabs/util-linux-tools` system extensions, which need to be installed before installing Longhorn.
+
+The most straightforward way is patching these system extensions into existing Talos Linux nodes.
+
+```yaml
+customization:
+  systemExtensions:
+    officialExtensions:
+      - siderolabs/iscsi-tools
+      - siderolabs/util-linux-tools
+```
+
+For detailed instructions, see Talos' documentation on [System Extensions](https://www.talos.dev/v1.6/talos-guides/configuration/system-extensions/) and [Boot Assets](https://www.talos.dev/v1.6/talos-guides/install/boot-assets/).
+
+## Pod Security
+
+Longhorn requires pod security `enforce: "privileged"`.
+
```

Comment:
Can we briefly explain why we need this? so users will better understand the context behind.